CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 2.2.0
+
+## Improvements
+
+* `ConfigurableItemsPattern`'s prop `itemsData` is no longer required
+
 # 2.1.0
```

Comment:
I don't think 2.1.0 has been released yet

Reply:
@SeanArmstrong TBH I wasn't sure where to add this. Shall I move it under the `2.1.0` heading then?

Reply:
yep